app/models/post.rb

```diff
@@ -15,4 +15,6 @@ def update_post_counter
   def five_most_recent_comments
     comments.order(updated_at: :desc).limit(5)
   end
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly add validation for `comments_counter` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=CommentsCounter%20must%20be%20an%20integer%20greater%20than%20or%20equal%20to%20zero.).